### PR TITLE
fix table path for Postgresql

### PR DESCRIPTION
**Pull request by Zaszczyk**

### CODE REVIEW

**Comment by sergeyklay** [comment]
We don't add changes directly to `master` branch. Can you please create PR to `2.0.x`?

File: scripts/Phalcon/Mvc/Model/Migration.php [hunk 1]
```diff
@@ -493,7 +493,13 @@ public static function migrate($fromVersion, $toVersion, $tableName)
      */
     public function morphTable($tableName, $definition)
     {
-        $defaultSchema = self::$_databaseConfig->get('dbname');
+        if(self::$_databaseConfig->get('adapter') === 'Postgresql'){
+            $defaultSchema = self::$_databaseConfig->get('dbname').'.'.self::$_databaseConfig->get('schema');
+        }
+        else{
```
Comment (sergeyklay) [inline scripts/Phalcon/Mvc/Model/Migration.php]: Use PSR-like style please
```php
if (..) {
} else { 
..
}
```

Comment (sergeyklay) [inline scripts/Phalcon/Mvc/Model/Migration.php]: See example of determining schema https://github.com/Zaszczyk/phalcon-devtools/blob/master/scripts/Phalcon/Mvc/Model/Migration.php#L191-L199

**Comment by Zaszczyk** [comment]
Sure, I did it here: https://github.com/phalcon/phalcon-devtools/pull/526

**Pull request closed without merging**